universal-application-tool-0.0.1/conf/evolutions/default/5.sql

```diff
@@ -0,0 +1,16 @@
+# --- Accounts
+
+# --- !Ups
+
+create table accounts (
+  id bigserial primary key
+  -- table does not contain a username or password hash yet.
+);
+
+alter table applicants add account bigint;
```

Comment:
suggestion (blocking): renaming `account` to `account_id` would make working with this column easier to read